index.js

```diff
@@ -0,0 +1,74 @@
+import luxon from './modules/luxon.js';
+import AwesomeBooks from './modules/books.js';
+
+const awesomeBooks = new AwesomeBooks();
+awesomeBooks.load();
+if (awesomeBooks.getBooks().length < 1) {
+  awesomeBooks.addBook('Double Standard 2', 'Dr. Samsul Arefin');
+}
+
+const bookListSection = document.querySelector('#book-list');
+
+const renderBookList = () => {
+  bookListSection.innerHTML = awesomeBooks
+    .getBooks()
+    .map(
+      (book, index) => `
+        <article class="book ${index % 2 === 0 ? 'dark' : ''}">
+            <div>
+                <p class="title">"${book.title}" by ${book.author}</p>
+            </div>
+            <button data-id=${book.id} class="remove">Remove</button>
+        </article>`,
+    )
+    .join('');
+};
+renderBookList();
+
+const addBookForm = document.querySelector('#add-book');
+addBookForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const title = event.target.querySelector('#title').value;
+  const author = event.target.querySelector('#author').value;
+  awesomeBooks.addBook(title, author);
+  event.target.reset();
+  renderBookList();
+});
+
+bookListSection.addEventListener('click', (event) => {
+  if (event.target.classList.contains('remove')) {
+    const { id } = event.target.dataset;
+    awesomeBooks.deleteBook(+id);
+    renderBookList();
+  }
+});
+
+const dateSection = document.querySelector('.date');
+const dateTime = luxon.DateTime.now().toLocaleString(luxon.DateTime.DATETIME_FULL);
+
+dateSection.innerHTML = dateTime;
+
+const bookList = document.querySelector('.list-link');
+const addNew = document.querySelector('.add-new');
+const contact = document.querySelector('.contact');
+const bookSection = document.querySelector('.books-list');
+const addNewSection = document.querySelector('.form');
+const contactSection = document.querySelector('.contact-container');
+
+bookList.addEventListener('click', () => {
+  bookSection.style.display = 'block';
+  addNewSection.style.display = 'none';
+  contactSection.style.display = 'none';
+});
+
+addNew.addEventListener('click', () => {
+  bookSection.style.display = 'none';
+  addNewSection.style.display = 'block';
+  contactSection.style.display = 'none';
+});
+
+contact.addEventListener('click', () => {
+  bookSection.style.display = 'none';
+  addNewSection.style.display = 'none';
+  contactSection.style.display = 'block';
+});
```

Comment:
- [ ] Great job implementing all your logic using ES6. One thing you can do differently is to have the add new section and contact section have `display: none` by default in your CSS file. That way, when the app loads, your contact and add book sections won't be shown initially like they are now. Kindly implement this feature.

![image](https://user-images.githubusercontent.com/90556221/183410601-ad3dff05-3342-43fc-8be5-ea661d6477a7.png)

Reply:
Thank you for your suggestion. I'll make your suggested changes.